tests/win32.spec.js

```diff
@@ -28,19 +29,17 @@ if (os.platform() === 'win32') {
 
       this.timeout(30000);
 
-      const diskpart = require('diskpart');
-
       describe('given diskpart always fails', function() {
```

Comment:
Could we change this description to: 'given the diskpart child-process always fails' ?